test/parallel/test-buffer.js

```diff
@@ -28,6 +28,10 @@ var c = new Buffer(512);
 console.log('c.length == %d', c.length);
 assert.strictEqual(512, c.length);
 
+var d = new Buffer([]);
+console.log('d.length == %d', d.length);
```

Comment:
Normally we don't expect successful tests to print anything. Can you remove this?